src/styles/blocks/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  &__item-img-link {
+    display: block;
+    margin-bottom: 40px;
+  }
+
+  &__item-img img {
+    display: block;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__item-name {
+    color: $title-text-color;
+    margin-bottom: 4px;
+
+    @include title-text-font;
+  }
+
+  &__item-code {
+    color: #616070;
+    margin-bottom: 16px;
+  }
+
+  &__item-review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__review a {
+    text-decoration: none;
+    color: $main-text-color;
+  }
+
+  &__item-price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__item-price-name {
+    font-size: 12px;
+    line-height: 18px;
+    color: #616070;
+  }
+
+  &__item-price-amount {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__btn-buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 40px;
+    border-radius: 5px;
+    border: none;
+    background-color: $blue-color;
+    cursor: pointer;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    color: $white-color;
+    text-decoration: none;
+    text-transform: uppercase;
+  }
+
+  &__btn-buy:hover {
+    border: 1px solid $blue-color;
+    background-color: $white-color;
+    color: $blue-color;
+  }
```

Comment:
you can nest it inside button class

Reply:
I don`t understand what I need to do here